### ISSUE TITLE
README says MNIST while in reality it's the 8x8 variant

### ISSUE THREAD

**Issue by darthdeus** [EpistasisLab/tpot#943]
The README gif says "~99% accuracy on MNIST out of the box", but this is not true. The model is being trained on https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html, which actually is not the 28x28 MNIST most people are using, but only an 8x8 reduced variant of it.

I ran into this trying to find a good classifier on the standard MNIST 28x28 dataset, not being able to get any good results (even with lots of compute).

I do believe this was done in good faith, and maybe coming from the deep learning side of things I'm biased in thinking that MNIST is 28x28, but I've spoken to multiple people and haven't found one who would think MNIST meant 8x8. I do realize that it is visible in the code in the gif, but personally, I never used the [`datasets.load_digits`](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html) function.

I did notice that [here in the README](https://github.com/EpistasisLab/tpot#classification) it says *practice MNIST*, which up until now I just translated as "MNIST dataset used for practice" and not "the practice version of MNIST".

**Comment by foxik** [comment]
Personally I think that when you say MNIST dataset you definitely do not mean the test portion of https://archive.ics.uci.edu/ml/datasets/Optical+Recognition+of+Handwritten+Digits dataset loaded by `load_digits` -- it returns 1797 digits in 8x8 resolution, compared to 60k+10k of 28x28 MNIST.

Furthermore, http://epistasislab.github.io/tpot/examples/ in the Overview says "MNIST" dataset, the dataset description links to the real MNIST https://yann.lecun.com/exdb/mnist/, and then there is link to Jupyter notebook https://github.com/EpistasisLab/tpot/blob/master/tutorials/MNIST.ipynb using the `load_digits` method.

**Comment by weixuanfu** [comment]
Thank you for pointing out this error in dataset descriptions. We will correct them in TPOT README and examples.

**Comment by weixuanfu** [comment]
The issue is fixed in new version of TPOT (v0.11.0). Please feel free to reopen it if you have any questions or suggestions.
